### PR TITLE
populate ee cmd to env

### PR DESCRIPTION
**Pull request by le-horizon**
This is needed to populate keybo controller's locomotion command (ee commands) to simulation env.

### CODE REVIEW

File: alf/environments/fast_parallel_environment.py [hunk 1]
```diff
@@ -188,6 +188,16 @@ def num_tasks(self):
     def task_names(self):
         return self._task_names
 
+    def set_commands(self, commands):
```
Comment (emailweixu) [inline alf/environments/fast_parallel_environment.py]: I suggest to add a generic call function to call each parallel environment with same arguments:
```
def call_with_same_args(self, func_name, *args, **kwargs);
```

Comment (le-horizon) [reply alf/environments/fast_parallel_environment.py]: Good point.  Done for both PRs.  PTAL.

File: alf/environments/process_environment.py [hunk 2]
```diff
@@ -377,6 +377,15 @@ def _receive(self):
         raise KeyError(
             'Received message of unexpected type {}'.format(message))
 
+    def set_commands(self, commands):
```
Comment (emailweixu) [inline alf/environments/process_environment.py]: Can be removed

Comment (le-horizon) [reply alf/environments/process_environment.py]: Done.

**Review by le-horizon — COMMENTED** [review]
Done, PTAL.

Thanks,
Le

**Review by emailweixu — APPROVED** [review]

**Pull request merged**